packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -241,12 +249,15 @@ export class WaferMap<
     public update(): void {
         this.validate();
         if (this.isExperimentalUpdate()) {
-            this.experimentalUpdate();
+            this.currentTask = this.experimentalUpdate().then(
+                () => {
+                    this.currentTask = undefined;
+                },
+                () => {}
+            );
```

Comment:
We should avoid using `.then` and prefer async / await

In this case can you just clear currentTask at the end of `experimentalUpdate`? make sure all the code paths clear it correctly

Reply:
sure! solved in this commit https://github.com/ni/nimble/pull/1929/commits/b1e08c9a676a5824cc848a4ad604cbf2659637c7